templates/cluster_resources.j2

```diff
@@ -168,10 +180,14 @@ primitive rsc_SAPHana_{{ sid }}_HDB{{ instance }} ocf:suse:SAPHana \
     op monitor interval="60" role="Master" timeout="700" \
     op monitor interval="61" role="Slave" timeout="700"
 
-ms msl_SAPHana_{{ sid }}_HDB{{ instance }} rsc_SAPHana_{{ sid }}_HDB{{ instance }} \
+ms msl_{{ SAPHanaResource }}_{{ sid }}_HDB{{ instance }} rsc_{{ SAPHanaResource }}_{{ sid }}_HDB{{ instance }} \
+{%- if scale_out %}
+    meta master-max="1" clone-node-max="1" interleave="true"
```

Comment:
`master-max` is alias for `promoted-max` and for backward compatibility, maybe/will be deprecated in future. So if the `scale_out` is not target for old version, maybe use `promoted-max` instead is better. 

Refer to https://www.clusterlabs.org/pacemaker/doc/deprecated/en-US/Pacemaker/2.0/html/Pacemaker_Explained/_clone_options.html for more details.

Reply:
This is based on https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-scaleOut-PerfOpt-12/index.html.
The old discussion... What shall be changed first? The docs? The code?

Reply:
Well, the doc still title as SLE12 SP2 or newer ... Even it has already end of lifecycle (https://www.suse.com/lifecycle)

So we can keep `master-max` here to align with docs.